omod/src/main/webapp/pages/metadata/privileges/privilege.gsp

```diff
@@ -55,35 +55,42 @@
     });
 </script>
 
-<h1>
-    <h3>${ ui.message((createPrivilege) ? "adminui.addNewPrivilege.label" : "adminui.editPrivilege.label")}</h3>
-</h1>
+<% if(privilege == null) {%>
+    <div align="center">
+        <h1>${ui.message('adminui.privilege.unknown')} "${unknownPrivilegeName}"</h1>
```

Comment:
Unencoded value introduces an XSS vulnerability here!

For example, try with privilege name `<script>alert(1)</script>`

Make sure to html-encode `unknownPrivilegeName`

Reply:
thanks @isears for the reminders 
![image](https://user-images.githubusercontent.com/58003327/122888097-32e76c00-d34a-11eb-899c-f6ae31459978.png)